src/FooterLinkManager.php

```diff
@@ -81,4 +81,14 @@ public function getSectionsAsOptions(): array {
     return $sections;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getListCacheTags(): array {
+    return array_merge(
+      $this->linkStorage->getEntityType()->getListCacheTags(),
+      $this->sectionStorage->getEntityType()->getListCacheTags()
+    );
+  }
```

Comment:
Let's remove this form here, it's not this service's scope to return cache tags of two entity type storage.

Reply:
Also mind the way tags should be merged, it's not done with an array merge, Drupal offers a service method for that.